files/build_templates/docker_image_ctl.j2

```diff
@@ -283,7 +283,7 @@ function postStartAction()
         fi
         REDIS_SOCK="/var/run/redis-chassis/redis_chassis.sock"
     fi
-    chgrp -f redis $REDIS_SOCK && chmod -f 0760 $REDIS_SOCK
+    chmod -f 0760 $REDIS_SOCK
```

Comment:
Why do we not need to chgrp for the redis sock?

Reply:
change PR direction to make redis process runs as non root and enable userns-remap, so no need to `chgrp -f redis $REDIS_SOCK`